MetaMask/metamask-extension
# feat:update konet image

## **Description**



Add KONET icon support. 

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/26244?quickstart=1)

## **Related issues**

Nothing

## **Manual testing steps**

1. Build
2. Add Konet Mainnet via Chainlist (https://chainlist.org/chain/17217)
4. See updated logo

## **Screenshots/Recordings**

(TBA)



### **Before**

(TBA)




### **After**

(TBA)



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
